crates/aranya-cli/testing/build-teams.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+# Create directories for all daemons
+for i in {1..5}; do
```

Comment:
Let's create a variable for the number of devices that can be used everywhere
```suggestion
for i in {1..${NUM_DEVICES}}; do
```